extensions/wikia/AchievementsII/services/AchAwardingService.class.php

```diff
@@ -491,42 +492,42 @@ private function processInTrackBlogComment() {
 
 	private function processInTrackLove() {
 		//BADGE_LOVE
-		if ( empty( $this->mCounters[BADGE_LOVE] ) ) {
-			$this->mCounters[BADGE_LOVE][COUNTERS_COUNTER] = 1;
+		if( empty( $this->mCounters[ BADGE_LOVE ] ) ) {
+			$this->mCounters[ BADGE_LOVE ][ COUNTERS_COUNTER ] = 1;
 		} else {
-			if ( $this->mCounters[BADGE_LOVE][COUNTERS_DATE] == date( 'Y-m-d' ) ) {
+			if( $this->mCounters[ BADGE_LOVE ][ COUNTERS_DATE ] == date( 'Y-m-d' ) ) {
 				// ignore
-			} else if ( $this->mCounters[BADGE_LOVE][COUNTERS_DATE] == date( 'Y-m-d', strtotime( '-1 day' ) ) ) {
-				$this->mCounters[BADGE_LOVE][COUNTERS_COUNTER]++;
+			} else if( $this->mCounters[ BADGE_LOVE ][ COUNTERS_DATE ] == date( 'Y-m-d', strtotime( '-1 day' ) ) ) {
+				$this->mCounters[ BADGE_LOVE ][ COUNTERS_COUNTER ]++;
 			} else {
-				$this->mCounters[BADGE_LOVE][COUNTERS_COUNTER] = 1;
+				$this->mCounters[ BADGE_LOVE ][ COUNTERS_COUNTER ] = 1;
 			}
 		}
-		$this->mCounters[BADGE_LOVE][COUNTERS_DATE] = date( 'Y-m-d' );
+		$this->mCounters[ BADGE_LOVE ][ COUNTERS_DATE ] = date( 'Y-m-d' );
 	}
 
 	private function incrementCounterBy( $badge_type_id, $value ) {
-		if ( !is_array( $this->mCounters ) ) {
+		if( !is_array( $this->mCounters ) ) {
 			throw new MWException( "Counters have to be initialized as array." );
 		}
 
-		if ( !isset( $this->mCounters[$badge_type_id] ) ) {
-			$this->mCounters[$badge_type_id] = 0;
+		if( !isset( $this->mCounters[ $badge_type_id ] ) ) {
+			$this->mCounters[ $badge_type_id ] = 0;
 		}
-		$this->mCounters[$badge_type_id] += $value;
+		$this->mCounters[ $badge_type_id ] += $value;
 	}
 
 	private function processAllNotInTrack() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this function "processAllNotInTrack" is 28 which is greater than 20 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1541)